From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 10 Jun 2021 22:20:47 -0700
Subject: [PATCH] Fix build with GCC 11

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 smb/autotests/transfertest.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/smb/autotests/transfertest.cpp b/smb/autotests/transfertest.cpp
index bdd4707..6f66adc 100644
--- a/smb/autotests/transfertest.cpp
+++ b/smb/autotests/transfertest.cpp
@@ -6,6 +6,7 @@
 #include <QTest>
 
 #include <future>
+#include <thread>
 
 #include "transfer.h"
 
